blog/2024-09-18-case-study-falai/index.md

```diff
@@ -0,0 +1,121 @@
+high-performance workloads. It’s designed to work seamlessly across clouds,
+countries, and continents, making it simple and efficient for developers.
+[Try it out, you’ll be up and running in a minute.](https://storage.new/)_
+
+<!-- truncate -->
+
+[fal.ai’s](https://fal.ai/) team set an ambitious goal: host the fastest
+diffusion inference endpoints in the world without passing the bill onto their
+users. Their platform needed to remain affordable for individual developers, all
+while ingesting 10s of TBs in mere hours, storing 100+ TBs of data around the
+globe, and offering [real time responses](https://fal.ai/docs/real-time).
+
+<span align="center">
+  ![Tigris loves fal.ai, generated by FLUX](falai.jpg)
+</span>
+
+## Inference faster than you can type
+
+Obsessed with optimization, fal.ai knows that every second counts on their GPUs.
+They upload output images in background threads so the only GPU time charged is
+for actual inference. Their researchers constantly test their production models
+against the state-of-the-art (SOTA) to find the best performing architectures
+for task precision, reliability, and reduced generation time. Every extra
+millisecond is shaved off, making fal.ai an order of magnitude more performant
+than their competitors for many generative ai tasks.
+
+> fal.ai has built custom infrastructure and optimized the model inference to
+> make sure these models are served to the end user as fast as possible. fal.ai
+> has a globally distributed network of GPUs to make sure the inference happens
+> as close to the user as possible. We do very little hops from between the user
+> and the GPU. —Burkay Gur, Co-founder fal.ai
+
+## Extreme performance
+
+Bringing the cutting-edge into industry requires translating both academic
+research and the latest community-driven workflows, into live, performant
+systems. And they mean performance— completely saturated 10Gb links, 1GB/s+
+writes, squeezing every last bit of juice from a global fleet of GPUs across
+many clouds. Every step from training to inference pipelines is carefully
+optimized and purpose built. Each network hop is scrutinized.
+
+[fal.ai](http://Fal.ai) tried other providers with no egress fees, but none of
+them met their reliability and performance needs. Low throughput, sluggish
+downloads, and intermittent 500 errors made it impossible to guarantee their
+diffusion endpoints could process requests in real time. After partnering with
+Tigris, fal.ai didn’t need to choose between performance, reliability, and cost.
+
+> have been using @TigrisData at @fal for the last 2 months. ingested 10s of TBs
+> of data in mere hours while storing 100 TB+ without any hassle. much much more
+> reliable than anything else we have used. and also MUCH FASTER. i am impressed
+> honestly. thanks @martin_casado for rec
+> —[Batuhan Taskaya](https://x.com/isidentical/status/1792631256586338349?s=46&t=uiM9vzvUExQcdhsrC5jgfg),
```

Comment:
Should we show this as an embedded tweet so that the tweet contents are also available?